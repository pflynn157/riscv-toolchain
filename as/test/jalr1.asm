jalr x12, x1, 8
jalr x14, x2, 4
