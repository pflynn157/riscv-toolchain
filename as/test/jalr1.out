00808667
00410767
